docs/markdown/Fs-module.md

```diff
@@ -29,3 +31,71 @@ name exists on the file system. This method follows symbolic links.
 
 Takes a single string argument and returns true if the path pointed to
 by the string is a symbolic link.
+
+## File Parameters
+
+### hash
+
+The `fs.hash(filename)` method computes the requested hash sum of a file.
+The available hash methods include: md5, sha1, sha224, sha256, sha384, sha512.
+
+### size
+
+The `fs.size(filename)` method returns the size of the file in bytes.
+Symlinks will be resolved if possible.
+
+### samefile
+
+The `fs.samefile(filename1, filename2)` method allows determining if two filenames refer to the same file.
+Perhaps a meson.build file in one place refer to a symlink and in another place a
+relative path and/or absolute path. The `samefile` method allows determining if these
+are the same file.
```

Comment:
There is no reason to single out `meson.build` by name here. It is in fact a bit confusing, because the common case is probably comparing files other than build files.

Reply:
Yes I agree